string init_state
string goal_state
---
bool solved
PddlAction[] plan
float32 cost